### DISCUSSION TITLE
Actividad 0 -Abregu, Candela

### DISCUSSION THREAD

**Discussion by CandelAbregu** [frlp-utn-ingsoft/catedra#84] [Show and tell]
https://github.com/CandelAbregu/NewActividad-0.git

**Comment by CandelAbregu** [comment]
Buen día Profe, disculpe la tardanza de la corrección de la actividad 0,  yo la volví a crear en otro repositorio pero en este caso puedo borrar el repositorio anterior o por las dudas que se mal interprete la fecha de entrega lo dejo?  Gracias

**Comment by RodrigoJacznik** [comment]
Buenas,
Los commits y sus mensajes están bien.
La app funciona.
La actividad esta aprobada.
